src/classes/ERR_Handler_API_TEST.cls

```diff
@@ -0,0 +1,106 @@
+/*
+    Copyright (c) 2014, Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2021
+* @description test class for Error Handling API
+* @group ErrorHandling
+*/
+@isTest
+private class ERR_Handler_API_TEST {
+
+    private static final String TEST_ACCOUNT_NAME = 'ERR_Handler_API Acct';
+    private static final String ERR_CONTEXT = 'ERR_Handler_API_TEST';
+
+    @TestSetup
+    static void setupInitalAccount() {
+        Account account = new Account(Name = TEST_ACCOUNT_NAME);
```

Comment:
Minor difference, but it's useful to use the test data factory. You also won't need `TEST_ACCOUNT_NAME` and the queries in the test methods can just select the first Account since there will always only be a single Account record in the system.
```suggestion
        Account account = UTIL_UnitTestData_TEST.createMultipleTestAccounts(1, null)[0];
```